installers/win-installer/pcgen.nsi

```diff
@@ -72,6 +73,18 @@ Icon "${SrcDir}\Local\PCGen2.ico"
 !insertmacro MUI_LANGUAGE "English"
 !insertmacro MUI_RESERVEFILE_LANGDLL
 
+!define ARP "Software\Microsoft\Windows\CurrentVersion\Uninstall\${APPDIR}"
+
+!macro VerifyUserIsAdmin
```

Comment:
A check for admin permissions.

Reply:
ideally we should not need this, but accepting for now